examples/wifi-echo/server/esp32/main/CHIPDeviceManager.cpp

```diff
@@ -67,6 +67,12 @@ CHIP_ERROR CHIPDeviceManager::Init(CHIPDeviceManagerCallbacks * cb)
         ConnectivityMgr().SetWiFiAPMode(ConnectivityManager::kWiFiAPMode_Enabled);
         break;
 
+    case RendezvousInformationFlags::kNone:
+        // If rendezvous is bypassed, enable SoftAP so that the device can still
+        // be communicated with via its SoftAP as needed.
+        ConnectivityMgr().SetWiFiAPMode(ConnectivityManager::kWiFiAPMode_Enabled);
```

Comment:
What about also enabling BLE with ```ConnectivityMgr().SetBLEAdvertisingEnabled(ConnectivityManager::kCHIPoBLEServiceMode_Enabled);```?
It will allow the chip-tool echo-ble command to continue to work even if Rendezvous is bypassed.

Reply:
I can do that. It wasn't clear to me whether we wanted that behavior, but it's easy to do.

Reply:
Currently on the controller side, BLE is specifically used for Secure Pairing and Rendezvous. Also, the controller does not have a Rendezvous bypass mode. Enabling BLE on device side won't help chip-tool's echo-ble command to work. But, maybe we should not enable echo-ble in chip-tool (and, instead integrate the Rendezvous flow with it).

Reply:
Given that, and slack dicussion about how to proceed here, I'm going to leave this PR as-is.